ha_cluster_exporter.go

```diff
@@ -183,34 +184,38 @@ func main() {
 
 	// set SBD device metrics
 	go func() {
-		for {
-			log.Println("[INFO]: Reading cluster SBD configuration..")
-			// read configuration of SBD
-			sbdConfiguration, err := readSdbFile()
-			if err != nil {
-				log.Println(err)
-				continue
-			}
-			// retrieve a list of sbd devices
-			sbdDevices := getSbdDevices(sbdConfiguration)
-			// set and return a map of sbd devices with true healthy, false not
-			sbdStatus := setSbdDeviceHealth(sbdDevices)
-
-			if len(sbdStatus) == 0 {
-				log.Println("[WARN]: Could not retrieve any sbd device")
-				continue
-			}
+		if _, err := os.Stat("/etc/sysconfig/sbd"); os.IsNotExist(err) {
```

Comment:
I like the check. The only thing which I would try to do is that if we could avodid if/else and just use guard clause.

e.g:

 _, err := os.Stat("/etc/sysconfig/sbd")
// check if file exist
if os.IsExist(err) {

// do sbd metrics
}


withing this we don't log however that we skip the `sbd` metric but in term of cognition overload is more simple to read

Reply:
That makes sense, pushed the suggested change. :slightly_smiling_face: